lib/views/components/custom_bottom_nav_bar.dart

```diff
@@ -2,11 +2,11 @@ import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:gocast_mobile/providers.dart';
 import 'package:gocast_mobile/views/course_view/pinned_courses_view/pinned_courses_view.dart';
-import 'package:gocast_mobile/views/notifications_view/notifications_screen_view.dart';
-
 import 'package:gocast_mobile/views/course_view/courses_overview.dart';
 import 'package:gocast_mobile/views/course_view/downloaded_courses_view/downloaded_courses_view.dart';
 
+import '../notifications_view/notifications_overview.dart';
```

Comment:
change to `package:gocast_mobile..........`

Reply:
Done ✅